phpBB/phpbb/path_helper.php

```diff
@@ -108,9 +108,13 @@ public function update_web_root_path($path)
 			$path = $this->phpbb_root_path . substr($path, strlen($web_root_path));
 		}
 
-		if (strpos($path, $this->phpbb_root_path) === 0)
+		if (strpos($path, $this->phpbb_root_path) === 0 || $this->phpbb_root_path === './')
```

Comment:
Not 100% happy with this but the current uses for update_web_root_path() are in append_sid() for relative paths and the assets for relative paths. The assets will be checked for allowed paths, append_sid() paths will not but are implied to be relative. I'll check what can be done there.